### DISCUSSION TITLE
Is there support for the NHD-C0216CiZ-FSW-FBW-3V3 LCD screen?

### DISCUSSION THREAD

**Discussion by jesus96x** [mattjlewis/diozero#204] [Q&A]
Hello, I was wondering if this library has support for this type of LCD display: https://newhavendisplay.com/content/specs/NHD-C0216CiZ-FSW-FBW-3V3.pdf.

I've been using a PCF8574/HD44780 LCD screen and has worked amazingly, but haven't been lucky with the NHD-C0216CiZ-FSW-FBW-3V3 one. I'm not super knowledgeable about electronics, so I'm sometimes guessing on what's the proper code to get a connection going. I wanted to ask if it's possible to draw something to this LCD with this library.

Thanks.

**Comment by mattjlewis** [comment]
I had a quick look at this device and didn't seem to be that easy to acquire. As you've found I don't think they are currently supported - look very similar to the HD44780 so should be easy to implement.
